types/spectest/run_test.go

```diff
@@ -79,12 +79,12 @@ func TestJson(t *testing.T) {
 				typedTest := &partialsigmessage.EncodingTest{}
 				require.NoError(t, json.Unmarshal(byts, &typedTest))
 				typedTest.Run(t)
-			case reflect.TypeOf(&share.EncodingTest{}).String():
-				byts, err := json.Marshal(test)
-				require.NoError(t, err)
-				typedTest := &share.EncodingTest{}
-				require.NoError(t, json.Unmarshal(byts, &typedTest))
-				typedTest.Run(t)
+			// case reflect.TypeOf(&share.EncodingTest{}).String():
+			// 	byts, err := json.Marshal(test)
+			// 	require.NoError(t, err)
+			// 	typedTest := &share.EncodingTest{}
+			// 	require.NoError(t, json.Unmarshal(byts, &typedTest))
+			// 	typedTest.Run(t)
```

Comment:
Since we are removing this test type, you can actually delete these lines :)

Reply:
You are right, removed